conceptual/Npgsql/diagnostics/tracing.md

```diff
@@ -29,3 +30,30 @@ For example, Zipkin visualizes traces in the following way:
 ![Zipkin UI Sample](/img/zipkin.png)
 
 In this trace, the Npgsql query (to database testdb) took around 800ms, and was nested inside the application's `work1` activity, which also had another unrelated `subtask1`. This allows understanding the relationships between the different activities, and where time is being spent.
+
+## Configuration options
+
+> [!NOTE]
+>
+> This feature was introduced in Npgsql 9.0
+
+Once you've enabled Npgsql tracing as above, you can tweak its configuration via the <xref:Npgsql.NpgsqlDataSourceBuilder.ConfigureTracingOptions*?displayProperty=nameWithType> API:
+
+```csharp
+dataSourceBuilder.ConfigureTracingOptions(new NpgsqlTracingOptions
+{
+    // Set the command SQL as the span name
+    ProvideSpanNameForCommand = cmd => cmd.CommandText,
+    // Filter out COMMIT commands
+    FilterCommand = cmd => !cmd.CommandText.StartsWith("COMMIT", StringComparison.Ordinal)
```

Comment:
`StringComparison.OrdinalIgnoreCase` is probably better?

Reply:
Yeah, typo, thanks :)